### ISSUE TITLE
[BUG] UIWebView is deprecated

### ISSUE THREAD

**Issue by kubarskii** [lamantin-group/react-native-yandex-payment#17]
Старая версия библиотеки yandex-checkout-payments-swift (2.2.0) использует UIWebView, из-за чего нельзя выложить приложение в app Store, можно ли обновить библиотеку до 3.0.0 или 3.1.0

**Comment by whalemare** [comment]
Да, я могу это сделать, но не быстро. Если вам нужно срочно, просто обновите зависимости и пришлите PR, я его приму

**Comment by mav10** [comment]
На сегодняшний день эта пробелма еще пристутсвует?

**Comment by kubarskii** [comment]
> На сегодняшний день эта пробелма еще пристутсвует?

Да, в новой версии библиотеки изменился интерфейс, также в последних версиях больше не используется trust defender
